Task6.py

```diff
@@ -0,0 +1,14 @@
+""" 6.Упорядоченный список.
+Дан список целых чисел. Требуется переместить
+все ненулевые элементы в левую часть списка, не меняя их порядок,
+а все нули - в правую часть. Порядок ненулевых элементов изменять
+нельзя,
+дополнительный список использовать нельзя, задачу нужно выполнить
+за один проход по списку.
+распечатайте полученный список."""
+scroll = [0, 1, 3, 0, 4, 5, 0]
+for k in range(0, len(scroll)):
+    if scroll[k] == 0:
+        del scroll[k]
```

Comment:
list.remove would be much more preferable

Reply:
well. thanks